vcd/resource_vcd_vapp_nat_rules_test.go

```diff
@@ -318,12 +324,22 @@ resource "vcd_vapp_vm" "{{.VmName2}}" {
 `
 
 const testAccVcdVappNatRules_rules = testAccVcdVappNatRules_vappAndVm + `
+# Must be defined just to enable FW for NAT rules to work if it wasn't enabled.
+# Just as exmample, doesn't impact test
```

Comment:
```suggestion
# Just as example, doesn't impact test
```
Please rephrase, because there is a contradiction between "Must be defined" in the line above and the assertions in this line.
Same for the comment at lines 387-388

Reply:
changed

Reply:
I prefer @lvirbalas suggestion
"In order to enable NAT service, firewall needs to be enabled as well"